packages/flutter_tools/bin/podhelper.rb

```diff
@@ -0,0 +1,136 @@
+# Copyright 2014 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Hook for Podfile setup, installation settings.
+#
+# @example
+# flutter_ios_podfile_setup
+# target 'Runner' do
+# ...
+# end
+def flutter_ios_podfile_setup
```

Comment:
Added this hook for future use (could have been used for the old `install! 'cocoapods', :disable_input_output_paths => true`, for example)